### PR TITLE
Reminders display a reminder on click closes #3

### LINKED ISSUE #3: Show an individual reminder
When the user clicks on one of the reminders on the page, they are taken to a `Reminder` route (e.g. "/1", if the id of the reminder is 1). The title of the reminder object should be displayed.

So, what needs to happen here?

- You'll need a `Reminder` route in addition to the `Reminders` route.
- It should be nested under the `Reminders` route.
- The user should still see all of the reminders from your previous feature. The specific reminder they are looking at now should be rendered in the outlet of the `reminders.hbs` template.
- There should be some CSS for the `.active` class.

There is a failing test set up in `tests/acceptance/reminder-list-test.js`.

### PR DESCRIPTION
**Pull request by Jeff-Duke**
## Purpose

Solved new acceptance test.  Reminders are now links, we are able to click on them and see the reminder rendered below the list of reminders. 

Installed Sass and applied styling to the active reminder link as well as the body of the reminder. 

## Approach

We added a new dynamic route for the individual reminder that looks for the reminder ID.  We also added a model hook for the reminder.  We setup the reminder template to render the title, date, and notes. 

### Learning
We reviewed some of the ember docs and also looked over beard-beats.

Paired with Brandon to get the test suite issues resolved.  Slacked the solution to our team (sah dude!)

### Test coverage 

Did not write any new tests but we did get the one failing test to pass.

@stevekinney @tehviking @martensonbj

### CODE REVIEW

File: app/styles/app.scss [hunk 1]
```diff
@@ -0,0 +1,13 @@
+@import 'reset.scss';
+
+ .active p{
```
Comment (stevekinney) [inline app/styles/app.scss]: Let's add a space before that open opening curly brace.

File: app/templates/reminders/reminder.hbs [hunk 2]
```diff
@@ -1,5 +1,14 @@
 <div class="reminder">
   <h3 class="spec-reminder-title">{{model.title}}</h3>
+
+  <p>
+    {{model.date}}
+  </p>
+
+  <p>
```
Comment (stevekinney) [inline app/templates/reminders/reminder.hbs]: This isn't going to block your PR, but eventually we're going to need to deal with the fact that there may not be notes and we won't want to show a `<p>` tag if there isn't any content.

**Comment by brittanystoroz** [comment]
Can we make the PR title a little more descriptive? It should be similar to a git commit message, telling you very briefly what the PR is solving. "Feature 2" isn't super helpful, it's fine to leave the "closes 3" if you are automating something with closing issues upon PR merges, but tell me what feature 2 is in the title. You can link to the issue or specs in the description.

File: app/templates/reminders/reminder.hbs (hunk 2 above)
Comment (Jeff-Duke) [inline app/templates/reminders/reminder.hbs]: Makes sense, we'll address it in the future

Comment (brittanystoroz) [inline app/templates/reminders/reminder.hbs]: Might be a good thing to file an issue for ;)

File: app/styles/app.scss (hunk 1 above)
Comment (Jeff-Duke) [inline app/styles/app.scss]: fixed

File: app/templates/reminders/reminder.hbs (hunk 2 above)
Comment (Jeff-Duke) [inline app/templates/reminders/reminder.hbs]: submitted issue #7

File: app/router.js [hunk 3]
```diff
@@ -7,7 +7,9 @@ const Router = Ember.Router.extend({
 });
 
 Router.map(function() {
-  this.route('reminders', {path: '/'});
+  this.route('reminders', { path: '/' }, function() {
+    this.route('reminder', { path: ':reminder_id'});
```
Comment (stevekinney) [inline app/router.js]: Add a space before that final closing curly brace for consistency.

Comment (Jeff-Duke) [inline app/router.js]: fixed 💅

**Pull request merged**